CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on
+[Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.1.0] - 2019-YY-ZZ
+
+### Changed
+
+- Supported version of Terraform is 0.12. [#6]
+
+## [2.0.0]
+
+## [0.1.0]
```

Comment:
same

Reply:
I don't see 0.2.0 release here. Please ping Aaron

Reply:
I will fix the CHANGELOG entries before we release.